File: Framework/wearable_health/lib/wearable_health_data_constants.dart
class WearableHealthDataConstants {
  WearableHealthDataConstants._();

  // Channel names
  static const String channelName = "se.lnu.thesis.wearable_health/methods";
  static const String eventChannelName = "se.lnu.thesis.wearable_health/events";

  // Method call names
  static const String methodGetPlatformVersion = "getPlatformVersion";
  static const String methodRequestPermissions = "requestPermissions";
  static const String methodStartCollecting = "startCollecting";
  static const String methodStopCollecting = "stopCollecting";
  static const String methodGetHealthData = "getHealthData";
}

File: Framework/wearable_health/android/src/main/kotlin/se/lnu/wearable_health/WearableHealthDataConstants.kt
package se.lnu.wearable_health

enum class WearableHealthDataConstants(val value: String) {
    // Channel names
    CHANNEL_NAME("se.lnu.thesis.wearable_health/methods"),
    EVENT_CHANNEL_NAME("se.lnu.thesis.wearable_health/events"),

    // Method names
    METHOD_GET_PLATFORM_VERSION("getPlatformVersion"),
    METHOD_REQUEST_PERMISSIONS("requestPermissions"),
    METHOD_START_COLLECTING("startCollecting"),
}